src/main/java/io/pinecone/clients/Pinecone.java

```diff
@@ -52,12 +82,14 @@ public IndexModel describeIndex(String indexName) throws PineconeException {
         return indexModel;
     }
 
-    public void configureIndex(String indexName, ConfigureIndexRequest configureIndexRequest) throws PineconeException {
+    public IndexModel configureIndex(String indexName, ConfigureIndexRequest configureIndexRequest) throws PineconeException {
+        IndexModel indexModel = new IndexModel();
```

Comment:
`configureIndex` actually returns an index, updated to handle that appropriately.

Reply:
We shouldn't need to construct it ahead of time -- you can just do:

```java
IndexModel indexModel;
```

Reply:
TIL ^ !

Reply:
Thank you! I think we should adjust all of these control plane functions given that rather than having all these empty objects that may possibly be returned.